crates/next-custom-transforms/src/transforms/warn_for_edge_runtime.rs

```diff
@@ -187,7 +187,8 @@ where
     EmitWarn: Fn(Span, String),
     EmitError: Fn(Span, String),
 {
-    fn warn_if_nodejs_module(&self, span: Span, module_specifier: &str) -> Option<()> {
+    fn warn_if_nodejs_module(&self, span: Span, module_specifier: &Wtf8Atom) -> Option<()> {
+        let module_specifier = module_specifier.as_str()?;
```

Comment:
Nits:

- We can use `Wtf8Atom`​'s `starts_with` to check for the `node:`​ prefix, skipping the potentially fallible conversion.
- We can do a lossy conversion when generating the error message.